### PR TITLE
Fixes #100 Implement Download as Zip functionality

### LINKED ISSUE #100: Enable "Download as Zip" functionality in the WebUI
### This is a
- [ ] Bug report
- [x] Feature request

Following up #85 

### Current behaviour

The WebUI only generates and stores the documentation without the functionality to access them.
### Expected behaviour

Enable "Download as Zip" functionality.

### PR DESCRIPTION
**Pull request by imujjwal96**
## Related Issue

Issue #100 
## How Has This Been Tested?

http://yaydoc.tk

## Types of changes

- [ ] Bug fix 
- [x] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

**Comment by imujjwal96** [comment]
@pri22296  @sch00lb0y  Please review

File: WebUI/generate.sh [hunk 1]
```diff
@@ -88,4 +83,11 @@ make html
 if [ $? -ne 0 ]; then
   echo -e "Failed to generate documentation.\n"
   exit 2
+fi
+
+mv $BUILD_DIR/_build/html $BUILD_DIR/../../${PROJECTNAME}_preview && cd $_/../
```
Comment (pri22296) [inline WebUI/generate.sh]: where is this being moved to? $BUILD_DIR can be at depth > 1. In that case i think, $BUILD_DIR/../../ would point to a different directory.

Comment (imujjwal96) [inline WebUI/generate.sh]: Moving it to temp/<email-id>/
Where the repository is cloned.

Comment (pri22296) [inline WebUI/generate.sh]: how can we be sure that $BUILD_DIR/../../ points to temp/<email-id>?

Comment (imujjwal96) [inline WebUI/generate.sh]: i got your point. Working on it.

Comment (imujjwal96) [inline WebUI/generate.sh]: @pri22296 i have updated it. Please review again.

**Pull request merged**